.github/workflows/release-published.yml

```diff
@@ -87,8 +87,11 @@ jobs:
         run: |
           if [ "${{ github.ref_name }}" == "release" ]; then
             echo "branch=release" >> $GITHUB_OUTPUT
-          else
+          elif [ "${{ github.ref_name }}" == "master" ]; then
             echo "branch=master" >> $GITHUB_OUTPUT
+          else
```

Comment:
Do we support releases from any branches other than release or master?

Reply:
No we dont.